supplier_management_site/locale/pt_BR/LC_MESSAGES/django.po

```diff
@@ -2301,73 +2301,81 @@ msgid "IIBB No Retention"
 msgstr ""
 
 #: supplier_app/models.py:300
-msgid "Certificate of no tax withholding of Tax liens, income or SUSS"
+#, fuzzy
+#| msgid "Certificate of no tax withholding of Tax liens, income or SUSS"
+msgid "Certificate of no income withholding"
 msgstr "Certificado sem retenção na fonte de IVA, renda ou SUSS"
```

Comment:
"Certificado sem retenção na fonte de renda"

Reply:
Portuguese translations have been halted since the product will launch only for Argentina for now, thats why most of it remains untranslated.
Still will apply those you have provided